### DISCUSSION TITLE
In object detecion is there any processing power difference if a small part of the image moves vs a multiple areas

### DISCUSSION THREAD

**Discussion by Cytomax55** [blakeblackshear/frigate#6481] [General]
I have a lot of plants/bushes close to my sidewalk and for me to competely get rid of background motion i would have to cover up the sidewalk

Am i helping reduce processing power by masking the lower 1/2 of the plant/bush but leaving the sidewalk and the top half of the plant exposed  and they have constant movement

OR

Am i better off not masking anything since i dont want to block 1/2 of my sidwalker

+++++++++++++++++++++++++++++++++++++++++++

Is there any processing power difference in object detection between

A single small area of motion in a frame

vs

Multiple areas of motion in the frame

++++++++++++++++++++++

I dont know if i am asking correctly if you are confused ill try to rephrase

**Comment by blakeblackshear** [comment]
You may want to try adjusting the settings under motion configuration rather than using a mask.
